SplunkRumWorkspace/SplunkRum/SplunkRum/SplunkRum.swift

```diff
@@ -408,4 +269,22 @@ var splunkRumInitializeCalledTime = Date()
     @objc public class func isInitialized() -> Bool {
         return initialized
     }
+
+    /**
+       Updates the current location. The latitude and longitude will be appended to every span and event.
+     */
+    @objc public class func setLocation(location: CLLocationManager) {
```

Comment:
For ease of use it should be `@objc public class func setLocation(latitude: Double, longitude: Double)`. Users might not want to use CLLocationManager, plus I think it's a bit more clear with explicit latitude and longitude (and reduces coupling).

Reply:
Okay, I'll change CLLocationManager to Double